common/actions/config.js

```diff
@@ -1,41 +1,45 @@
 // @flow
+import {
+  NODE_CHANGE,
+  LANGUAGE_CHANGE,
+  GAS_PRICE_CHANGE
+} from './configConstants';
 
 export type ChangeNodeAction = {
-  type: 'CONFIG_NODE_CHANGE',
+  type: NODE_CHANGE,
```

Comment:
Ah, I see what it is. It's because all of these actions had the same keys, and then each action was split into functions in the reducers, so there was a conflict. One action expected value to be a `number`, and the other a `string`.

The other issue is that by splitting each action into its own function, they have inconsistent inputs. The _only_ reason this worked before was they were all an identical layout, `{ type: string, value: string }`. If we added a different action signature, it would have broken the whole thing.

To fix this, I'm going to move everything back into the reducer function, and stop using a generic `value` key, to prevent these kinds of convenient collisions.